src/components/Header/index.js

```diff
@@ -87,18 +59,20 @@ const Header = ({
               `}
             >
               <li>
-                <NavLink to="/dashboard">
+                <NavLink currentPathName={currentPathName} to="/dashboard">
                   <HouseIcon /> <Trans>Dashboard</Trans>
                 </NavLink>
               </li>
-              <li>
-                <ToSearchPage index="file" css={theme.navLink}>
-                  <DatabaseIcon /> <Trans>File Repository</Trans>
-                </ToSearchPage>
-              </li>
+              {indeces.map(index => (
```

Comment:
since this is such a finite list I think I'd prefer to see it just declared (no array iteration). you'd have to account for the `File Repository` thing below anyway